deploy/metrics-apiservice.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1alpha1.metrics
+spec:
+  service:
+    name: metrics-server
+    namespace: kube-system
+  group: metrics
+  version: v1alpha1
+  insecureSkipTLSVerify: true
+  groupPriorityMinimum: 100
```

Comment:
Are these two values actually the recommended ones? IIRC one of them should be higher